app/controllers/application_controller.rb

```diff
@@ -1,2 +1,17 @@
 class ApplicationController < ActionController::API
```

Comment:
After changes to how `UsersController#create` works. Do we need all of those methods?

Reply:
no, we do not need them